2019-07-04 21:40:11,666 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBBB5A20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/3e6c4001a165747d2c328f8081a7eb02
2019-07-04 21:40:12,672 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CB91A4A8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/3e6c4001a165747d2c328f8081a7eb02
2019-07-04 21:40:13,667 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBB90160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/3e6c4001a165747d2c328f8081a7eb02
2019-07-04 21:41:32,805 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-04 21:46:49,663 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-04 21:48:52,033 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-04 21:51:00,200 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-04 21:52:31,656 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-04 21:53:59,812 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-04 21:55:02,068 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-04 21:55:13,638 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-04 21:55:23,798 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-04 21:55:43,329 - INFO - register_us.py[:236] - AnnacVaylea701@zaful.biz用户注册成功！
2019-07-04 21:55:46,166 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-04 21:55:49,365 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CB930F98>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ac9ecff4dc4c3d9516834bd4ee17218a
2019-07-04 21:55:50,367 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBD68A20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ac9ecff4dc4c3d9516834bd4ee17218a
2019-07-04 21:55:51,380 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBD68198>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ac9ecff4dc4c3d9516834bd4ee17218a
2019-07-04 22:01:02,078 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:02:09,749 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-04 22:02:32,863 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 22:14:31,200 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:20:03,536 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:25:21,276 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:26:37,693 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-04 22:28:39,450 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-04 22:28:49,478 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-04 22:29:04,981 - INFO - register_us.py[:236] - MaryanncCassa670@zaful.work用户注册成功！
2019-07-04 22:29:08,728 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-04 22:29:13,047 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBB90780>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/39c9425c1e0f981c442f7869f3702a40
2019-07-04 22:29:14,131 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBD4C9B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/39c9425c1e0f981c442f7869f3702a40
2019-07-04 22:29:15,148 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CB8FBF98>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/39c9425c1e0f981c442f7869f3702a40
2019-07-04 22:30:32,911 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-04 22:31:20,936 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-04 22:31:33,336 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-04 22:31:43,483 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-04 22:31:56,884 - INFO - register_us.py[:236] - HenrettacGlenne267@zaful.fun用户注册成功！
2019-07-04 22:32:09,385 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-04 22:32:13,514 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBD68588>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/fe75dab63ef9f1489b65de1d9d50d892
2019-07-04 22:32:14,519 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBBB5358>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/fe75dab63ef9f1489b65de1d9d50d892
2019-07-04 22:32:15,516 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBBB5240>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/fe75dab63ef9f1489b65de1d9d50d892
2019-07-04 22:34:03,217 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-04 22:39:16,487 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:44:32,862 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:49:45,563 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-04 22:51:29,751 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-04 22:56:01,333 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-04 22:56:12,583 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-04 22:56:22,666 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-04 22:56:40,032 - INFO - register_us.py[:236] - EarthacRickeyc498@ninebot.ltd用户注册成功！
2019-07-04 22:56:42,998 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-04 22:56:47,498 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBBB5DA0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/305f3ffe3483944122059abcf40a39c4
2019-07-04 22:56:48,496 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBD4CFD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/305f3ffe3483944122059abcf40a39c4
2019-07-04 22:56:49,497 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000068CBD4C4E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/305f3ffe3483944122059abcf40a39c4
2019-07-04 22:56:53,495 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 23:07:01,505 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 23:17:09,375 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 23:27:13,958 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 23:37:26,098 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 23:47:30,127 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-04 23:57:33,844 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 00:07:37,621 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 00:17:42,971 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 00:27:49,976 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 00:37:52,337 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 00:48:15,221 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 00:58:20,900 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 01:08:29,207 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 01:18:33,500 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 01:28:36,685 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 01:38:40,501 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 01:48:45,206 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 01:58:49,728 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 02:08:53,716 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 02:18:57,525 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 02:29:04,271 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 02:39:07,580 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 02:49:10,572 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 02:59:20,338 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 03:09:24,324 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 03:19:34,285 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 03:29:37,463 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 03:39:40,506 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 03:49:45,420 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 03:59:48,888 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 04:09:52,506 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 04:19:54,378 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 04:29:57,293 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 04:40:18,118 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 04:50:21,153 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 05:00:30,749 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 05:10:33,215 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 05:20:36,089 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 05:30:38,772 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 05:40:41,971 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 05:50:45,242 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 06:00:56,596 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 06:11:01,008 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 06:21:04,899 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 06:31:08,091 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 06:41:13,055 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 06:51:16,589 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 07:01:19,540 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 07:11:22,836 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 07:21:26,162 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 07:31:29,504 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 07:41:32,354 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 07:51:35,364 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 08:01:44,779 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 08:11:58,742 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 08:22:01,535 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 08:32:05,568 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 08:42:10,570 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 08:52:13,778 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 09:02:22,615 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 09:12:26,881 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 09:22:30,943 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 11:56:38,257 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 12:06:44,242 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 12:16:47,600 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 12:26:51,022 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 12:36:54,303 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 12:46:58,458 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 12:57:02,285 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 13:07:04,393 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 13:17:07,548 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 13:27:12,814 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 13:37:16,251 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 13:47:19,593 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 13:57:22,874 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 14:07:29,124 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 14:17:32,013 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 14:27:34,450 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 14:37:38,496 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 14:47:48,373 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 14:57:51,387 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 15:07:54,854 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 15:17:57,526 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 15:28:01,134 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 15:38:03,680 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 15:48:05,741 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 15:58:09,302 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 16:08:15,333 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 16:18:20,052 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 16:28:22,160 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 16:38:25,737 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 16:48:29,674 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 16:58:32,798 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 17:08:35,641 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 17:17:37,193 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 17:27:58,701 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 17:38:01,403 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 17:48:06,386 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 17:58:16,247 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 18:08:18,007 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 18:18:22,013 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 18:29:42,754 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 18:31:12,536 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 18:32:38,612 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:33:46,992 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:33:57,025 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:34:12,890 - INFO - register_us.py[:236] - JudithaLamiae976@aukey.live用户注册成功！
2019-07-05 18:34:14,451 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:34:17,594 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410860>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/2bcfe7cf473c381504fd95538df87c3e
2019-07-05 18:34:18,597 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D400>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/2bcfe7cf473c381504fd95538df87c3e
2019-07-05 18:34:19,602 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/2bcfe7cf473c381504fd95538df87c3e
2019-07-05 18:35:28,041 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:36:13,118 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 18:36:22,517 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:36:32,551 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:36:45,245 - INFO - register_us.py[:236] - BedroseLeife804@aukey.fun用户注册成功！
2019-07-05 18:36:55,331 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:36:58,453 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D4012B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e58f7fe313675db42ac5b877f2fd8743
2019-07-05 18:36:59,479 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E95C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e58f7fe313675db42ac5b877f2fd8743
2019-07-05 18:37:00,476 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/e58f7fe313675db42ac5b877f2fd8743
2019-07-05 18:38:39,249 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:40:44,514 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:40:54,585 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:41:10,387 - INFO - register_us.py[:236] - UlyssaeFilmorec915@aukey.ltd用户注册成功！
2019-07-05 18:41:19,358 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:41:22,559 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647860>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/29fc753972653394d2f2ff16a70fb2e9
2019-07-05 18:41:23,659 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647048>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/29fc753972653394d2f2ff16a70fb2e9
2019-07-05 18:41:24,662 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647DD8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/29fc753972653394d2f2ff16a70fb2e9
2019-07-05 18:43:09,543 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 18:44:26,263 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:45:22,315 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:45:32,463 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:45:46,426 - INFO - register_us.py[:236] - AvrameRonc143@aukey.net.cn用户注册成功！
2019-07-05 18:45:48,654 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:45:54,248 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622A58>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5698c1143be14a1969c3b17807b0cfb2
2019-07-05 18:45:55,247 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622668>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5698c1143be14a1969c3b17807b0cfb2
2019-07-05 18:45:56,245 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D6226D8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5698c1143be14a1969c3b17807b0cfb2
2019-07-05 18:47:12,374 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:48:00,907 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 18:48:09,164 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:48:19,216 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:48:33,309 - INFO - register_us.py[:236] - ElamaKevonc209@aukey.mobi用户注册成功！
2019-07-05 18:48:34,522 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:48:38,635 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410128>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bbaa42b947f029f66efefa0273d3da57
2019-07-05 18:48:39,639 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9208>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bbaa42b947f029f66efefa0273d3da57
2019-07-05 18:48:40,633 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9860>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bbaa42b947f029f66efefa0273d3da57
2019-07-05 18:49:55,259 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:51:41,807 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:51:51,842 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:52:07,996 - INFO - register_us.py[:236] - LoueZackarea624@nineboys.co用户注册成功！
2019-07-05 18:52:16,205 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:52:20,012 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DF28>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/572a3f8f706814c81fba4d4dadd5ee6e
2019-07-05 18:52:21,022 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D0F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/572a3f8f706814c81fba4d4dadd5ee6e
2019-07-05 18:52:22,023 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D668>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/572a3f8f706814c81fba4d4dadd5ee6e
2019-07-05 18:53:37,862 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 18:54:19,776 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 18:54:28,051 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 18:54:38,084 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 18:54:50,344 - INFO - register_us.py[:236] - TommieeGilona283@nineboys.store用户注册成功！
2019-07-05 18:54:53,460 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 18:54:56,566 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622AC8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/38dfeb3e1efd5bfdaee189026d362eae
2019-07-05 18:54:57,571 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CBF28>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/38dfeb3e1efd5bfdaee189026d362eae
2019-07-05 18:54:58,578 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D4100B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/38dfeb3e1efd5bfdaee189026d362eae
2019-07-05 18:58:41,404 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 18:59:58,056 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 19:05:11,592 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 19:06:27,060 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:07:23,550 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 19:07:33,579 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 19:07:46,662 - INFO - register_us.py[:236] - SandersaBaronec162@nineboys.art用户注册成功！
2019-07-05 19:07:49,126 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 19:07:52,229 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/646769aef66ac10f3bf7b7629d8bc3b7
2019-07-05 19:07:53,249 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D0B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/646769aef66ac10f3bf7b7629d8bc3b7
2019-07-05 19:07:54,245 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D400>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/646769aef66ac10f3bf7b7629d8bc3b7
2019-07-05 19:13:01,990 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 19:14:25,500 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 19:19:35,451 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 19:21:30,499 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 19:22:56,889 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:24:41,250 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 19:24:51,296 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 19:25:06,841 - INFO - register_us.py[:236] - DesiraecSheryla327@nineboys.site用户注册成功！
2019-07-05 19:25:09,237 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 19:25:13,422 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622F98>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/00cc1435e30690b24afd4735d2f47335
2019-07-05 19:25:14,432 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/00cc1435e30690b24afd4735d2f47335
2019-07-05 19:25:15,434 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CBA20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/00cc1435e30690b24afd4735d2f47335
2019-07-05 19:26:42,420 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 19:28:36,909 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:29:54,733 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 19:30:04,768 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 19:30:21,889 - INFO - register_us.py[:236] - BardaeDarinae621@nineboys.pro用户注册成功！
2019-07-05 19:30:24,583 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 19:30:31,519 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65FDA0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/63c4cc663f28ee2f5a4745f6c822565e
2019-07-05 19:30:32,619 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D6229E8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/63c4cc663f28ee2f5a4745f6c822565e
2019-07-05 19:30:33,691 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622C88>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/63c4cc663f28ee2f5a4745f6c822565e
2019-07-05 19:32:22,500 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:33:22,054 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 19:33:29,440 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 19:33:39,474 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 19:33:51,909 - INFO - register_us.py[:236] - ZoeyaTabana391@nineboys.org用户注册成功！
2019-07-05 19:33:52,867 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 19:33:56,964 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65FA90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5319b40bb8b40347edae6bbe637f8c05
2019-07-05 19:33:57,967 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401080>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5319b40bb8b40347edae6bbe637f8c05
2019-07-05 19:33:58,972 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5319b40bb8b40347edae6bbe637f8c05
2019-07-05 19:39:08,625 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 19:40:28,915 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 19:45:37,866 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 19:46:41,131 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:47:27,424 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 19:47:33,205 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 19:47:43,253 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 19:47:55,548 - INFO - register_us.py[:236] - BarnetteAshlinga904@nineboys.info用户注册成功！
2019-07-05 19:47:56,642 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 19:48:02,031 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9c907a86dc37c8140effcbaca81206b7
2019-07-05 19:48:03,146 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410FD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9c907a86dc37c8140effcbaca81206b7
2019-07-05 19:48:04,336 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E93C8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9c907a86dc37c8140effcbaca81206b7
2019-07-05 19:49:11,951 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:49:59,735 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 19:50:05,483 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 19:50:15,513 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 19:50:27,801 - INFO - register_us.py[:236] - BennettcPadroa468@nineboys.top用户注册成功！
2019-07-05 19:50:28,897 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 19:50:32,006 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65F6A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/50eddc14681ef8204e60ef3f695c5d79
2019-07-05 19:50:33,005 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622198>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/50eddc14681ef8204e60ef3f695c5d79
2019-07-05 19:50:34,008 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622470>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/50eddc14681ef8204e60ef3f695c5d79
2019-07-05 19:52:07,032 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 19:53:08,976 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 19:58:17,843 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:00:24,891 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:02:16,394 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 20:02:26,423 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 20:02:46,836 - INFO - register_us.py[:236] - KeyonaFairyc612@nineboys.tech用户注册成功！
2019-07-05 20:02:49,627 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 20:02:53,753 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DF60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6a434fca768a59075f0f0b5e2e72d737
2019-07-05 20:02:54,759 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DB70>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6a434fca768a59075f0f0b5e2e72d737
2019-07-05 20:02:55,762 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D9E8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6a434fca768a59075f0f0b5e2e72d737
2019-07-05 20:04:17,405 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:05:21,132 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 20:06:55,203 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 20:08:20,761 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 20:13:30,704 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:19:05,239 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:21:00,828 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:22:26,324 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 20:22:36,355 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 20:22:53,561 - INFO - register_us.py[:236] - EnricoeRachelc371@nineboys.ltd用户注册成功！
2019-07-05 20:22:54,795 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 20:22:58,939 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D5F8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9531dd33dd0713c9067321ede72bfa7d
2019-07-05 20:22:59,946 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DB00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9531dd33dd0713c9067321ede72bfa7d
2019-07-05 20:23:00,947 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DD30>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9531dd33dd0713c9067321ede72bfa7d
2019-07-05 20:24:12,320 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 20:25:29,892 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:26:35,524 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 20:26:45,553 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 20:26:58,427 - INFO - register_us.py[:236] - DorieaSerenaa657@nineboys.group用户注册成功！
2019-07-05 20:26:59,703 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 20:27:03,851 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DEF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9b985d33ed83c0dd1bf548eb5e1ca466
2019-07-05 20:27:04,854 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D7B8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9b985d33ed83c0dd1bf548eb5e1ca466
2019-07-05 20:27:05,860 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D5C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/9b985d33ed83c0dd1bf548eb5e1ca466
2019-07-05 20:28:43,885 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 20:30:23,316 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 20:34:55,524 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:35:49,984 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 20:36:00,057 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 20:36:14,413 - INFO - register_us.py[:236] - ErhardcAnabellaa392@nineboys.xyz用户注册成功！
2019-07-05 20:36:29,181 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 20:36:32,312 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ea939b7df7d3845ea5a16e8b563ceffd
2019-07-05 20:36:33,315 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622470>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ea939b7df7d3845ea5a16e8b563ceffd
2019-07-05 20:36:34,318 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622CC0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ea939b7df7d3845ea5a16e8b563ceffd
2019-07-05 20:41:45,601 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:43:13,653 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:48:23,112 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:53:33,146 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 20:55:00,261 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:56:01,980 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 20:56:09,715 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 20:56:19,743 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 20:56:33,719 - INFO - register_us.py[:236] - NorberteManleya471@nineboys.cn用户注册成功！
2019-07-05 20:56:34,871 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 20:56:39,007 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ba1ec6832aba71fe912d3280899ea52c
2019-07-05 20:56:40,017 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647EF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ba1ec6832aba71fe912d3280899ea52c
2019-07-05 20:56:41,026 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622C18>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ba1ec6832aba71fe912d3280899ea52c
2019-07-05 20:58:11,088 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 20:59:26,808 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 20:59:31,771 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 21:14:40,689 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 21:14:50,817 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-05 21:25:57,090 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 21:27:15,110 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 21:28:21,689 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 21:28:31,722 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 21:28:45,860 - INFO - register_us.py[:236] - AnatolecTamae302@nineboys.club用户注册成功！
2019-07-05 21:28:47,025 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 21:28:50,165 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/00001fabb639c6008856e5ba4c7525b2
2019-07-05 21:28:51,165 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65FB38>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/00001fabb639c6008856e5ba4c7525b2
2019-07-05 21:28:52,172 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622198>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/00001fabb639c6008856e5ba4c7525b2
2019-07-05 21:30:11,757 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 21:31:39,072 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 21:36:48,115 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 21:38:24,575 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 21:40:29,984 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 21:41:52,614 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 21:43:39,903 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 21:44:52,601 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 21:45:02,633 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 21:45:19,378 - INFO - register_us.py[:236] - NormandcEmiliee874@nineboys.fun用户注册成功！
2019-07-05 21:45:21,348 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 21:45:24,577 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401E10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/1832f085d8f94a962ddfc44bea88e72f
2019-07-05 21:45:25,580 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/1832f085d8f94a962ddfc44bea88e72f
2019-07-05 21:45:26,592 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65FA20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/1832f085d8f94a962ddfc44bea88e72f
2019-07-05 21:46:44,643 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 21:47:32,862 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 21:47:40,779 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 21:47:50,853 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 21:48:05,387 - INFO - register_us.py[:236] - RagnaraJoya854@nineboy.club用户注册成功！
2019-07-05 21:48:07,021 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 21:48:10,230 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410320>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/23aa2ca62af7e6da2b03a9afb24909f0
2019-07-05 21:48:11,236 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/23aa2ca62af7e6da2b03a9afb24909f0
2019-07-05 21:48:12,255 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9278>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/23aa2ca62af7e6da2b03a9afb24909f0
2019-07-05 21:49:45,144 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 21:50:59,140 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 21:51:09,184 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 21:51:23,533 - INFO - register_us.py[:236] - JaneeDalite947@nineboy.co用户注册成功！
2019-07-05 21:51:24,917 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 21:51:28,059 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65FC18>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ec08ad9741b09e3ee99c113fd517a780
2019-07-05 21:51:29,063 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410518>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ec08ad9741b09e3ee99c113fd517a780
2019-07-05 21:51:30,067 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D410470>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ec08ad9741b09e3ee99c113fd517a780
2019-07-05 21:56:39,162 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 21:58:00,130 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 21:59:02,721 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 21:59:12,773 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 21:59:28,238 - INFO - register_us.py[:236] - AvrameLeanna973@nineboy.store用户注册成功！
2019-07-05 21:59:34,125 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 21:59:37,278 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65F860>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d7c963560db66583647a2843b67b187d
2019-07-05 21:59:38,273 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CBE10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d7c963560db66583647a2843b67b187d
2019-07-05 21:59:39,277 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CB780>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d7c963560db66583647a2843b67b187d
2019-07-05 22:00:59,470 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:03:21,841 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 22:05:08,238 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 22:05:18,282 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 22:05:57,598 - INFO - register_us.py[:236] - XerxescJilla753@nineboy.group用户注册成功！
2019-07-05 22:05:59,194 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 22:06:03,311 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CBBE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5e4a5cacf157f26c6289f4cd37abfc42
2019-07-05 22:06:04,315 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CBE10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5e4a5cacf157f26c6289f4cd37abfc42
2019-07-05 22:06:05,320 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CBB00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/5e4a5cacf157f26c6289f4cd37abfc42
2019-07-05 22:07:40,237 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:12:57,859 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 22:18:10,158 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 22:19:58,880 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 22:20:59,224 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 22:22:25,508 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:24:01,167 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:29:16,357 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 22:30:34,292 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 22:31:17,295 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 22:31:24,912 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 22:31:34,974 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 22:31:48,511 - INFO - register_us.py[:236] - MollyeTravea864@nineboy.ltd用户注册成功！
2019-07-05 22:31:52,105 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 22:31:55,221 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3BF056D8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/21dc72d203544dc2acf73eeea1a1d6e4
2019-07-05 22:31:56,231 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D8D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/21dc72d203544dc2acf73eeea1a1d6e4
2019-07-05 22:31:57,238 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62D668>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/21dc72d203544dc2acf73eeea1a1d6e4
2019-07-05 22:35:45,975 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 22:37:17,583 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:40:44,648 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 22:44:48,716 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 22:46:30,107 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:47:39,637 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 22:49:22,299 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 22:50:32,264 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 22:51:24,845 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 22:51:34,889 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 22:51:47,729 - INFO - register_us.py[:236] - StormcNyssac845@nineboy.art用户注册成功！
2019-07-05 22:51:49,165 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 22:51:52,324 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401CF8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/1fe8483bbab3e5eb6a5c0ca7de10b0f9
2019-07-05 22:51:53,355 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401748>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/1fe8483bbab3e5eb6a5c0ca7de10b0f9
2019-07-05 22:51:54,370 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E95C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/1fe8483bbab3e5eb6a5c0ca7de10b0f9
2019-07-05 22:53:52,331 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 22:56:15,742 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 22:56:51,333 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 22:57:01,380 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 22:57:36,531 - INFO - register_us.py[:236] - IaineDyane054@geabest.club用户注册成功！
2019-07-05 22:57:40,267 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 22:57:43,409 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DF60>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bc50c14cb890129887774749fc6eb2be
2019-07-05 22:57:44,426 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D401BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bc50c14cb890129887774749fc6eb2be
2019-07-05 22:57:45,441 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D4013C8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/bc50c14cb890129887774749fc6eb2be
2019-07-05 23:02:42,685 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 23:05:41,899 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 23:10:53,488 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 23:12:57,504 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 23:14:08,070 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 23:15:26,426 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 23:16:21,918 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 23:16:31,951 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 23:16:45,594 - INFO - register_us.py[:236] - EleanorcHoldene924@geabest.ltd用户注册成功！
2019-07-05 23:17:00,650 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 23:17:07,213 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65F390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/efac62c164240c19c17a8739c2e8b61a
2019-07-05 23:17:08,229 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622780>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/efac62c164240c19c17a8739c2e8b61a
2019-07-05 23:17:09,258 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D62DA58>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/efac62c164240c19c17a8739c2e8b61a
2019-07-05 23:18:34,459 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 23:23:45,125 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 23:26:01,998 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 23:28:52,819 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 23:30:17,205 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 23:34:47,195 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 23:35:37,155 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 23:35:47,207 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 23:35:59,649 - INFO - register_us.py[:236] - ChristyaHewettc156@geabest.info用户注册成功！
2019-07-05 23:36:05,174 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 23:36:08,345 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f47e6729d9cd8f4f7041b7b7f689ef48
2019-07-05 23:36:09,439 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E95C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f47e6729d9cd8f4f7041b7b7f689ef48
2019-07-05 23:36:10,456 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3E9160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/f47e6729d9cd8f4f7041b7b7f689ef48
2019-07-05 23:37:39,568 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 23:42:42,628 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-05 23:44:18,998 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 23:45:42,295 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 23:47:09,745 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-05 23:48:29,182 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 23:49:46,967 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-05 23:55:03,807 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-05 23:56:27,896 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-05 23:57:19,264 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-05 23:57:29,299 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-05 23:57:42,345 - INFO - register_us.py[:236] - DevinaMinorc652@geabest.pro用户注册成功！
2019-07-05 23:57:47,046 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-05 23:57:50,184 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65FF28>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/edc45d4ddb51cfa575d0f718aca28a50
2019-07-05 23:57:51,192 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D65F978>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/edc45d4ddb51cfa575d0f718aca28a50
2019-07-05 23:57:52,197 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647390>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/edc45d4ddb51cfa575d0f718aca28a50
2019-07-05 23:59:28,951 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-06 00:04:40,468 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-06 00:10:13,235 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-06 00:15:27,110 - INFO - register_us.py[:149] - 等待超时，程序终止！
2019-07-06 00:22:11,879 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-06 00:26:01,049 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-06 00:28:20,136 - INFO - register_us.py[:145] - 注册出错，程序结束！
2019-07-06 00:30:01,279 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-06 00:31:32,819 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-06 00:34:38,605 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-06 00:34:48,644 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-06 00:35:41,212 - INFO - register_us.py[:236] - YanessaeBirdiea759@geabest.mobi用户注册成功！
2019-07-06 00:35:45,886 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-06 00:35:50,258 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647710>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d54a4579815f1c31973acab5badd913a
2019-07-06 00:35:51,353 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3E4AF208>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d54a4579815f1c31973acab5badd913a
2019-07-06 00:35:52,353 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3E4AF780>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/d54a4579815f1c31973acab5badd913a
2019-07-06 00:37:46,495 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-06 00:38:55,383 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-06 00:39:10,619 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-06 00:39:21,429 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-06 00:39:39,423 - INFO - register_us.py[:236] - HerthacKamrone160@geabest.com.cn用户注册成功！
2019-07-06 00:39:44,326 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-06 00:39:48,216 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3E4AF1D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ac7cfedee4e37aedb4654b15e88c4b12
2019-07-06 00:39:49,219 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D3CB6D8>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ac7cfedee4e37aedb4654b15e88c4b12
2019-07-06 00:39:50,221 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3E49E940>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/ac7cfedee4e37aedb4654b15e88c4b12
2019-07-06 00:41:38,217 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-06 00:44:01,660 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-06 00:45:54,820 - INFO - register_us.py[:215] - 从接口获取的StateOrRegion信息对应不上，默认为下拉列表的第一个州或地区！
2019-07-06 00:46:07,534 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-06 00:46:17,563 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-06 00:46:57,965 - INFO - register_us.py[:236] - TabithacBlanee710@geabest.cc用户注册成功！
2019-07-06 00:46:59,493 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-06 00:47:02,701 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3E4AFFD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6cf9bd0a2d4b0510c7ad258a5dd30b2e
2019-07-06 00:47:03,704 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3E4AFE48>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6cf9bd0a2d4b0510c7ad258a5dd30b2e
2019-07-06 00:47:04,707 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D647D30>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/6cf9bd0a2d4b0510c7ad258a5dd30b2e
2019-07-06 00:48:51,899 - INFO - register_us.py[:126] - 注册出错，程序结束！
2019-07-06 00:50:35,800 - INFO - register_us.py[:154] - 正在设置用户的基本信息...
2019-07-06 00:51:51,187 - INFO - register_us.py[:231] - 点击提交个人信息的设置!
2019-07-06 00:52:01,242 - INFO - register_us.py[:117] - 个人信息设置成功！success!
2019-07-06 00:52:16,158 - INFO - register_us.py[:236] - VidalcDialloe156@geabest.online用户注册成功！
2019-07-06 00:52:22,228 - INFO - register_us.py[:270] - 注册信息已成功入库！
2019-07-06 00:52:26,451 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3DEA6898>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/7df89dfa03fb596d5b75c4ca0fe43433
2019-07-06 00:52:28,072 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D6226A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/7df89dfa03fb596d5b75c4ca0fe43433
2019-07-06 00:52:29,473 - WARNING - connectionpool.py[:662] - Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000004A3D622D68>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it',)': /session/7df89dfa03fb596d5b75c4ca0fe43433
2019-07-06 00:52:33,915 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 01:02:42,217 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 01:12:48,438 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 01:23:11,990 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 01:33:15,939 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 01:43:19,512 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 01:53:22,646 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 02:03:25,276 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 02:13:27,870 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 02:23:31,381 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 02:33:35,812 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 02:43:37,788 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 02:53:42,482 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 03:04:03,724 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 03:14:07,505 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 03:24:10,370 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 03:34:14,581 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 03:44:18,343 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 03:54:42,193 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 04:04:45,600 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 04:14:55,348 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 04:24:58,858 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 04:35:02,272 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 04:45:05,717 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 04:55:10,414 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 05:05:13,768 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 05:15:16,641 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 05:25:21,126 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 05:35:24,383 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 05:45:33,020 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 05:55:40,064 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 06:05:44,009 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 06:15:48,589 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 06:25:58,655 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 06:36:01,654 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 06:46:04,165 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 06:56:07,192 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 07:06:11,266 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 07:16:13,170 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 07:26:32,223 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 07:37:12,089 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 07:47:14,457 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 07:57:16,963 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 08:07:20,907 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 08:17:24,729 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 08:27:28,368 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 08:37:33,925 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 08:47:38,868 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 08:57:49,923 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 09:07:58,170 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 09:18:20,684 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 09:28:23,339 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 09:38:26,102 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 09:48:28,861 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 09:58:48,698 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 10:08:51,514 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 10:18:54,645 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 10:28:57,712 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 10:39:01,078 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 10:49:04,769 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 10:59:09,235 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 11:09:11,855 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 11:19:14,134 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 11:29:16,185 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 11:39:18,080 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 11:49:48,556 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 11:59:51,587 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 12:09:59,805 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 12:20:03,318 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 12:30:12,673 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 12:40:17,424 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 12:50:19,287 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 13:00:22,572 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 13:10:34,069 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 13:20:39,240 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 13:30:44,144 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 13:40:50,924 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 13:50:53,389 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 14:00:56,343 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 14:11:00,888 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 14:21:02,433 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 14:31:04,826 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 14:42:42,926 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 14:51:58,568 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 15:02:01,449 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 15:12:07,086 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 15:22:09,437 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 15:32:13,862 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 15:42:16,754 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 15:52:26,692 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 16:02:50,555 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 16:13:03,258 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 16:23:18,092 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 16:33:40,594 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 16:43:43,464 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 16:53:47,295 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 17:03:50,425 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 17:13:53,575 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 17:23:56,945 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 17:33:59,630 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 17:44:02,626 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 17:54:06,798 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 18:04:17,289 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 18:14:20,779 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 18:24:22,057 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 18:34:28,257 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 18:44:30,736 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 18:56:29,405 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 19:06:32,369 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 19:16:34,877 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 19:26:41,251 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 19:36:43,390 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 19:46:45,016 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 19:56:47,742 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
2019-07-06 20:06:51,192 - INFO - register_us.py[:344] - 当前无注册任务，程序[sleep 10m]...
